interface/main/tabs/menu/menus/standard.json

```diff
@@ -16,7 +16,19 @@
     ]
   },
   {
```

Comment:
When this file is ready, also should copy it to sites/default/documents/custom_menus/Custom.json
(this is basically a copy of standard.json, which folks can use as a starting point to create their own custom menus)